update_formspec.lua

```diff
@@ -78,136 +80,137 @@ travelnet.update_formspec = function( pos, puncher_name, fields )
 		"label[0.3,1.2;"..S("Owned by:").."]"..
 		"label[6.3,1.2;"..minetest.formspec_escape(owner_name or "?").."]"..
 		"label[3.3,1.6;"..S("Click on target to travel there:").."]"..
-		zusatzstr;
+		zusatzstr
 
-	local x = 0;
-	local y = 0;
-	local i = 0;
+	local x = 0
+	local y = 0
+	local i = 0
 
 
 	-- collect all station names in a table
-	local stations = {};
+	local stations = {}
 
 	for k in pairs( travelnet.targets[ owner_name ][ station_network ] ) do
-		table.insert( stations, k );
+		table.insert( stations, k )
 	end
 
-	local ground_level = 1;
+	local ground_level = 1
 	if is_elevator then
-		table.sort( stations, function(a,b) return travelnet.targets[ owner_name ][ station_network ][ a ].pos.y >
-		travelnet.targets[ owner_name ][ station_network ][ b ].pos.y  end);
+		table.sort( stations, function(a,b)
+			return travelnet.targets[ owner_name ][ station_network ][ a ].pos.y > travelnet.targets[ owner_name ][ station_network ][ b ].pos.y
+		end)
```

Comment:
To reduce the line length here, variables should be made:
```suggestion
		table.sort( stations, function(a,b)
			local y_a = travelnet.targets[owner_name][station_network][a].pos.y
			local y_b = travelnet.targets[owner_name][station_network][b].pos.y
			return y_a > y_b
		end)
```

Reply:
I really don't want to be changing things in the PR, I will just add the newline in around the `>` comparator, I will take care of that kind of thing in another PR later as there are lots of cases of variables  that could be simplified.
Personally I think this could be written even better as
```
		table.sort(stations, function(a, b)
			local network =  travelnet.targets[owner_name][station_network]
			return network[a].pos.y > network[b].pos.y
		end)
```
Could even define that outside, like
```
		local network =  travelnet.targets[owner_name][station_network]
		table.sort(stations, function(a, b)
			return network[a].pos.y > network[b].pos.y
		end)
```
But that's beyond linting